src/kernels/ADMatCoupledDefectAnnihilation.C

```diff
@@ -0,0 +1,52 @@
+/********************************************************/
+/*             DO NOT MODIFY THIS HEADER                */
+/* TMAP8: Tritium Migration Analysis Program, Version 8 */
+/*                                                      */
+/*    Copyright 2021 Battelle Energy Alliance, LLC      */
+/*               ALL RIGHTS RESERVED                    */
+/********************************************************/
+
+#include "ADMatCoupledDefectAnnihilation.h"
+
+// MOOSE includes
+#include "MooseVariable.h"
+#include "NonlinearSystem.h"
+
+registerMooseObject("MooseApp", ADMatCoupledDefectAnnihilation);
+
+InputParameters
+ADMatCoupledDefectAnnihilation::validParams()
+{
+  InputParameters params = ADKernel::validParams();
+  params.addCoupledVar("v",
+                       "Set this to make v a coupled variable, otherwise it will use the "
+                       "kernel's nonlinear variable for v");
+  params.addClassDescription(
+      "Kernel to add K*v*(c0-c), where K=annihilation rate, u=variable, v=coupled variable");
```

Comment:
Would prefer swapping this so that the class description leads the list with the rest of the params/vars following, so that the description is up front.

Reply:
Sounds good, I also updated the notation from `c` to `u` to be consistent